packages/wiggle/src/__snapshots__/WiggleRendering.test.js.snap

```diff
@@ -7,7 +7,7 @@ exports[`one 1`] = `
   style="overflow: visible; position: relative; height: 500px;"
 >
   <canvas
-    data-testid="prerendered_canvas"
+    data-testid="prerendered_canvas_undefined"
```

Comment:
could be worth checking if the undefined here is unintended

Reply:
The WiggleRendering.test and PileupRendering.test had empty config objects which lead to undefined type. Since I newly appended config.type to the testid, config was not needed to pass in previously for the test. I have now added a config object with a type: DummyRenderer to the test call so undefined should no longer appear. Does not look like an issue outside of test files, have not seen any surprise 'undefined' with the config.type